templates/search-results.hbs

```diff
@@ -3,113 +3,135 @@
                     <p>DECORATION</p>
                 </div>
             {{/contains}}
             {{#contains this.keywords "library"}}
-                <div class="library-badge result-badge">
+                <div class="library-badge result-badge" title="Library">
                     <img alt="library" src="/static/images/icon/library.svg" class="result-image"/>
                     <p>LIBRARY</p>
                 </div>
             {{/contains}}
             {{#contains this.keywords "worldgen"}}
-                <div class="world-badge result-badge">
+                <div class="world-badge result-badge" title="World">
                     <img alt="world" src="/static/images/icon/world.svg" class="result-image"/>
                     <p>WORLDGEN</p>
                 </div>
             {{/contains}}
             {{#contains this.keywords "cursed"}}
-                <div class="cursed-badge result-badge">
+                <div class="cursed-badge result-badge" title="Cursed">
                     <img alt="cursed" src="/static/images/icon/cursed.png" class="result-image"/>
                     <p>CURSED</p>
                     <p>CURSED</p>
                 </div>
             {{/contains}}
             {{#contains this.keywords "storage"}}
-                <div class="storage-badge result-badge">
+                <div class="storage-badge result-badge" title="Storage">
                     <img alt="storage" src="/static/images/icon/storage.svg" class="result-image"/>
                     <p>STORAGE</p>
                 </div>
             {{/contains}}
             {{#contains this.keywords "food"}}
-                <div class="food-badge result-badge">
+                <div class="food-badge result-badge" title="Food">
                     <img alt="food" src="/static/images/icon/food.svg" class="result-image"/>
                     <p>FOOD</p>
                 </div>
             {{/contains}}
             {{#contains this.keywords "equipment"}}
-                <div class="equipment-badge result-badge">
+                <div class="equipment-badge result-badge" title="Equipment">
                     <img alt="equipment" src="/static/images/icon/equipment.svg" class="result-image"/>
                     <p>EQUIPMENT</p>
                 </div>
             {{/contains}}
             {{#contains this.keywords "misc"}}
-                <div class="misc-badge result-badge">
+                <div class="misc-badge result-badge" title="Misc">
                     <img alt="misc" src="/static/images/icon/misc.svg" class="result-image"/>
                     <p>MISC</p>
                 </div>
             {{/contains}}
         </div>
     </div>
+{{else}}
+    <div class="search-error">
+        {{#if ../query.q}}
+        <h2>No results found for query <code>"{{../query.q}}"</code></h2>
+        {{else}}
+        <h2>No results found</h2>
+        {{/if}}
+    </div>
```

Comment:
This part is super great but if the search query is long enough the div will overflow. Not a big problem though, but should be an easy fix
![Screenshot_20200625_175347](https://user-images.githubusercontent.com/18202329/85809472-da689900-b70c-11ea-912a-b18a545190cd.png)